apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +253,78 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+}: {
+  csrfToken: string;
+  initialStartDateFormatted?: string;
+  initialEndDateFormatted?: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create Course Instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="short_name">CIID (Course Instance Identifier)</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          title="Please enter a valid format: only letters, numbers, hyphens, underscores, backslashes, and forward slashes."
+        />
+        <small class="form-text text-muted"
+          >The recommended format is <code>Fa19</code> or <code>Fall2019</code>. Use only letters,
+          numbers, dashes, and underscores, with no spaces.
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="long_name">Long Name</label>
+        <input type="text" class="form-control" id="long_name" name="long_name" required />
+        <small class="form-text text-muted">
+          This is the full name of the course instance, such as "Fall 2019" or "Spring 2020".
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="start_access_date">Start Access Date</label>
+        <input
```

Comment:
Adding `step="1"` should help ensure that the input always shows with second resolution.